Frameworks/Starboard/AssetFile.cpp

```diff
@@ -343,6 +343,31 @@ int EbrStat(const char* filename, struct stat* ret) {
     return stat(map.MappedPath(), ret);
 }
 
+int EbrStat64i32(const char* filename, struct _stat64i32* ret) {
+    CPathMapper map(filename);
+    char* fixedName = map.FixedPath();
+    if (!fixedName) {
+        return -1;
+    }
+
+    char* assetName = fixedName + 1;
+
+    if (EbrIsDir(filename)) {
+        memset(ret, 0, sizeof(struct _stat64i32));
+        ret->st_size = 0;
+        ret->st_mode = 0x1B6 | 0040000;
+        // MurmurHash3_x86_32(assetName, strlen(assetName), 0, &ret->st_ino);
+        return 0;
```

Comment:
We were faking inodes for .. some .. reason?

---
In reply to: [83125758](https://github.com/Microsoft/WinObjC/pull/1137#discussion_r83125758) [](ancestors = 83125758)